### PR TITLE
update macos plist contents (fixes #132 / #137)

### LINKED ISSUE #132: "PrismLauncher" on Mac menu bar
Should be named "Prism Launcher" instead of "PrismLauncher" on the MacOS menu bar.

CleanShot 2022-10-19 at 21 29 51@2x

_Originally posted by @tobimori in https://github.com/PrismLauncher/PrismLauncher/issues/126#issuecomment-1284476930_

### PR DESCRIPTION
**Pull request by tobimori**
This should fix the issues mentioned in #132.

Besides that, the copyright string should also be changed, as macOS doesn't recognize /n as new line.
CleanShot 2022-10-22 at 15 02 05@2x

I suggest it should be replaced with a hyphen, but this would need a change of the global variable that defines the copyright or a variable specific for macOS only, so it might be a good idea to specify how you'd address that further.

### CODE REVIEW

**Comment by DioEgizio** [comment]
> Besides that, the copyright string should also be changed, as macOS doesn't recognize /n as new line

Yeah, this has been an issue since polymc times

**Comment by DioEgizio** [comment]
also, you have to update .github/workflows/trigger_release.yml  too

**Comment by tobimori** [comment]
> also, you have to update .github/workflows/trigger_release.yml too

Workflow should be updated, although I didn't find any specific mention of PrismLauncher.app in the file you mentioned.

**Comment by DioEgizio** [comment]
oh yeah

**Comment by tobimori** [comment]
Seems like this wasn't everything. Somewhere, cmake assumes the .app name is still prismlauncher.app, right?

**Comment by Scrumplex** [comment]
Maybe you can revert the rename of the .app file. We can just rename the .app in the release process.

**Comment by tobimori** [comment]
> Maybe you can revert the rename of the .app file. We can just rename the .app in the release process.

will do. any opinions on the copyright string or leave it for now?

File: CMakeLists.txt [hunk 1]
```diff
@@ -222,14 +222,14 @@ if(UNIX AND APPLE)
...
-    set(MACOSX_BUNDLE_INFO_STRING "${Launcher_Name}: A custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once.")
+    set(MACOSX_BUNDLE_BUNDLE_NAME "${Launcher_DisplayName}")
+    set(MACOSX_BUNDLE_INFO_STRING "${Launcher_DisplayName}: A custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once.")
     set(MACOSX_BUNDLE_GUI_IDENTIFIER "org.prismlauncher.${Launcher_Name}")
     set(MACOSX_BUNDLE_BUNDLE_VERSION "${Launcher_VERSION_NAME}")
     set(MACOSX_BUNDLE_SHORT_VERSION_STRING "${Launcher_VERSION_NAME}")
     set(MACOSX_BUNDLE_LONG_VERSION_STRING "${Launcher_VERSION_NAME}")
     set(MACOSX_BUNDLE_ICON_FILE ${Launcher_Name}.icns)
-    set(MACOSX_BUNDLE_COPYRIGHT "Copyright 2021-2022 ${Launcher_Copyright}")
+    set(MACOSX_BUNDLE_COPYRIGHT "© 2021-2022 ${Launcher_Copyright_Mac}")
```
Comment (txtsd) [inline CMakeLists.txt]: This should only be 2022

**Review by txtsd — APPROVED** [review]
LGTM

**Review by Scrumplex — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**